includes/core/abstracts/form.php

```diff
@@ -221,12 +258,12 @@ public function set_post_meta_settings() {
 
 		/* one-time payment options */
 
-		$this->amount = simpay_convert_amount_to_cents( simpay_get_filtered( 'amount', simpay_get_saved_meta( $this->id, '_amount', '100' ), $this->id ) );
+		$this->amount = simpay_get_filtered( 'amount', simpay_get_saved_meta( $this->id, '_amount', simpay_global_minimum_amount() ), $this->id );
```

Comment:
This should be double checked.

Reply:
Looks correct. Part of amount conversion refactoring done in early 2018 in Pro.
https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/commit/fc7474f73217f19868c735b2b78d67e8568a98e2#diff-e560c0ff2a130a6267e5401b99a13c37